src/QMCWaveFunctions/Fermion/MultiDiracDeterminant.2.cpp

```diff
@@ -187,69 +285,113 @@ void MultiDiracDeterminant::BuildDotProductsAndCalculateRatios(int ref,
     }
 #endif
 }
+//[[deprecated]]
+void MultiDiracDeterminant::mw_BuildDotProductsAndCalculateRatios(int nw,
```

Comment:
Do we need this dummy function to compile the code? if not, remove.
If the single walker one is also needed, remove it as well.